front-end/src/App.vue

```diff
@@ -72,17 +71,6 @@ const resourcesPrefetch = ref<Array<Resource> | undefined>(undefined);
 
 const searchListContainer = ref<HTMLDivElement | null>(null);
```

Comment:
If we're removing the ref we should remove the searchListContainer variable and it's corresponding ref binding on the search-list-container div (Line 25!)